code/modules/mob/living/simple_animal/hostile/mining_mobs/ice demon.dm

```diff
@@ -85,3 +85,17 @@
 	if(prob(5))
 		new /obj/item/raw_anomaly_core/bluespace(loc)
 	return ..()
+
+
+//SKYRAT CHANGE, NERFED ICE DEMON FOR MINING
+/mob/living/simple_animal/hostile/asteroid/ice_demon/mining
+	name = "lesser demonic watcher"
+	icon_state = "ice_demon_lesser"
```

Comment:
instead of only making them smaller; also make them blue-green so as to better differentiate